server/bundles/io.cloudbeaver.server/schema/service.events.graphqls

```diff
@@ -95,6 +95,13 @@ type CBTopicEvent implements CBClientEvent {
     topicId: CBEventTopic! # topic to subscribe / unsubscribe
 }
 
+# Project event
+type CBProjectEvent implements CBClientEvent {
+    id: CBClientEventId!
+    topicId: CBEventTopic
+    projectId: String!
+}
```

Comment:
`CBClientEvent` sent by web application to the server
please use `implements CBServerEvent` and `id: CBServerEventId!`